cherry_picker/cherry_picker.py

```diff
@@ -636,7 +636,7 @@ def continue_cherry_pick(self):
             ]
             self.commit_sha1 = get_full_sha_from_short(short_sha)
 
-            commits = get_commits_from_backport_branch(base)
+            commits = get_commits_from_backport_branch(f"{self.upstream_remote}/{base}")
```

Comment:
`self.upstream` seems appropriate here, since `self.upstream_remote` is set to `None` if the command line option is not specified.

Reply:
Makes sense, thanks!
```suggestion
            commits = get_commits_from_backport_branch(f"{self.upstream}/{base}")
```